lib/marquee.dart

```diff
@@ -278,6 +279,21 @@ class Marquee extends StatefulWidget {
   ///   how the transition between moving and pausing state occur.
   final Duration pauseAfterRound;
 
+  /// When the text a rounded X times, it will stop scrolling
+  /// 0 is default value and is the value for the infinite loop
+  ///
+  /// ## Sample code
+  ///
+  /// After every round, this marquee pauses for one second.
+  ///
+  /// ```dart
+  /// Marquee(
+  ///   stopAfterMaxRound:3,
+  ///   text: 'Pausing for some time after every round.'
+  /// )
+  /// ```
+  final int stopAfterMaxRound;
```

Comment:
I'm not entirely happy with the name of the parameter.
- It's multiple rounds, so we should use plural.
- And _stop_ is an action (imperative) while _max rounds_ is a definition (declarative). The parameter is declarative, so it probably shouldn't contain the verb "stop".

I think `numberOfRounds` would be a better name. Is that okay with you?